### ISSUE TITLE
The influxdb2 cli 2.7.1-2 package requires Z Standard

### ISSUE THREAD

**Issue by 3BK** [influxdata/influxdb#24202]
__Steps to reproduce:__
Attempt to install the influxdb2 cli 2.7.1-2 package on an operating system which doesn't recognize Z Standard (i.e. RFC 8878) 
For example any verison of Redhat earlier than RHEL or CentOS 8.2.

yum update -y influxdb2-cli

Loaded plugins: langpacks, product-id, ps, search-disabled-repos, subscription-manager
Resolving Dependencies
--> Running transaction check
---> Package influxdb2-cli.x86_64  old will be updated
---> Package influxdb2-cli.x86_64 0:2.7.1-2 will be an update
--> Finished Dependency Resolution

Dependencies Resolved

 Package                                                 Arch                                             Version                                             Repository                                            Size

Updating:
 influxdb2-cli                                           x86_64                                           2.7.1-2                                             influxdata                                           9.6 M

Transaction Summary

Upgrade  1 Package

Total size: 9.6 M
Downloading packages:
Running transaction check
ERROR You need to update rpm to handle:
rpmlib(PayloadIsZstd) <= 5.4.18-1 is needed by influxdb2-cli-2.7.1-2.x86_64
RPM needs to be updated
 You could try running: rpm -Va --nofiles --nodigest
Your transaction was saved, rerun it with:
 yum load-transaction /tmp/yum_save_tx.2023-04-17.15-07.O2xA_D.yumtx

__Expected behavior:__
Describe what you expected to happen.
successfull installation.

__Actual behavior:__
Describe What actually happened.
rpmlib(PayloadIsZstd) <= 5.4.18-1 is needed by influxdb2-cli-2.7.1-2.x86_64

1st attempt at a workaround:
yum install zstd
Loaded plugins: langpacks, product-id, ps, search-disabled-repos, subscription-manager
No package zstd available.
Error: Nothing to do

__Environment info:__

* System info: Run `uname -srm` and copy the output here
Available upon request

* InfluxDB version: Run `influxd version` and copy the output here
InfluxDB v2.7.0 (git: 85f725f8b9) build_date: 2023-04-05T15:32:25Z

* Other relevant environment details: Container runtime, disk info, etc

__Config:__
Copy any non-default config values here or attach the full config as a gist or file.

__Logs:__
Include snippet of errors in log.

__Performance:__
Generate profiles with the following commands for bugs related to performance, locking, out of memory (OOM), etc.

```sh
# Commands should be run when the bug is actively happening.
# Note: This command will run for ~30 seconds.
curl -o profiles.tar.gz "http://localhost:8086/debug/pprof/all?cpu=30s"
iostat -xd 1 30 > iostat.txt
# Attach the `profiles.tar.gz` and `iostat.txt` output files.
```

**Comment by jeffreyssmith2nd** [comment]
Thanks for the report, this should be fixed in the next release of the CLI.

**Comment by 3BK** [comment]
np.  I'm probably going to upgrade the OS - regardless. ;)

**Comment by jeffreyssmith2nd** [comment]
This should be addressed in the 2.7.3 release of the CLI. Please open a new issue if you encounter any further issues!

**Comment by 3BK** [comment]
Noted.  Thanks Jeffery
